Tasks/IISWebAppDeploymentOnMachineGroup/task.json

```diff
@@ -0,0 +1,147 @@
+{
+  "id": "1B467810-6725-4B6D-ACCD-886174C09BBA",
+  "name": "IISWebAppDeploymentOnMachineGroup",
+  "friendlyName": "IIS Web App Deploy On Machine Group",
+  "description": "Deploy a WebSite or Virtual Application by using MSDeploy on Machine Group",
+  "helpMarkDown": "[More Information](https://aka.ms/iiswebappdeployreadme)",
+  "category": "Deploy",
+  "visibility": [
+    "Preview",
+    "Build",
+    "Release"
+  ],
+  "runsOn" : [
+  "MachineGroup"
+  ],
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 0,
+    "Minor": 1,
+    "Patch": 2
+  },
+  "demands": [
+  ],
+  "minimumAgentVersion": "2.104.1",
```

Comment:
The min agent version requirement of 2.104.1 is required only if you are enabling variable substitution. You may not need this otherwise.